core/src/test/scala/org/http4s/client/jdkhttpclient/CompletableFutureTerminationTest.scala

```diff
@@ -18,23 +18,23 @@ package org.http4s.client.jdkhttpclient
 
 import cats.syntax.all._
 import cats.effect._
-import cats.effect.concurrent._
 import java.util.concurrent.TimeUnit
 import scala.concurrent._
 import scala.concurrent.duration._
 import org.http4s._
 import org.http4s.server._
 import org.http4s.server.blaze._
 import cats.data._
-import cats.effect.testing.specs2.CatsIO
+import cats.effect.testing.specs2.CatsEffect
```

Comment:
Probably more than you want to bite off, partiicularly in this PR, but I'm not regretting any of our moves toward munit in the main repo.

Reply:
Ah, thanks for reminding me. This is also a good idea as e.g. `ClientRouteTestBattery` are also munit-based.